config.go

```diff
@@ -192,19 +188,95 @@ func loadConfiguration(configPath string) (oci.RuntimeConfig, ShimConfig, error)
 
 			config.ProxyType = vc.CCProxyType
 			config.ProxyConfig = pConfig
+
+			break
+		}
+	}
+
+	for k, agent := range tomlConf.Agent {
+		switch k {
+		case hyperstartAgent:
+			path := filepath.Join(agent.pauseBundlePath(),
+				pauseBinRelativePath)
```

Comment:
Well I didn't want to add this "rootfs" inside the code, I think it is better to have it from the absolute bundle path we are providing from configuration.toml